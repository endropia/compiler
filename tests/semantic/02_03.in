var
	i: integer;
begin
	for i := 0 to 0.1 do begin end;
end.