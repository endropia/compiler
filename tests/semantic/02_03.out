(4, 16) Integer expected